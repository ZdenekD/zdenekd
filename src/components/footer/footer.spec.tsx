import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '../../__test__/utils/helpers';
import Footer from '.';

describe('Footer', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Footer />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-footer');

        expect(component.exists()).toBe(true);
    });
});
