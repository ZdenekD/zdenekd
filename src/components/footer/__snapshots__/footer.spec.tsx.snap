// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer match snapshot 1`] = `
<footer class="footer closed"
        data-test="component-footer"
        style="opacity:0"
>
  <code class="code">
    ♣ MMXXI Vyrobeno pomocí &lt;kouzel&gt; a {zaklínadel}
  </code>
  <div class="linkWrapper">
    <a href="mailto:ditezdenek&amp;#64;pm.me?subject=&amp;clubs; Zpráva přímo do schránky &amp;clubs;"
       title="Zpráva přímo do schránky"
       class="link"
    >
      <svg width="28"
           height="28"
           viewbox="0 0 24 24"
           stroke-width="1"
           stroke="currentColor"
           fill="none"
           stroke-linecap="round"
           stroke-linejoin="round"
      >
        <path stroke="none"
              d="M0 0h24v24H0z"
              fill="none"
        >
        </path>
        <path d="M4 21v-13a3 3 0 0 1 3 -3h10a3 3 0 0 1 3 3v6a3 3 0 0 1 -3 3h-9l-4 4">
        </path>
        <line x1="8"
              y1="9"
              x2="16"
              y2="9"
        >
        </line>
        <line x1="8"
              y1="13"
              x2="14"
              y2="13"
        >
        </line>
      </svg>
    </a>
    <a href="https://github.com/ZdenekD"
       title="Github"
       class="link"
       target="_blank"
       rel="noreferrer noopener"
    >
      <svg width="28"
           height="28"
           viewbox="0 0 24 24"
           stroke-width="1"
           stroke="currentColor"
           fill="none"
           stroke-linecap="round"
           stroke-linejoin="round"
      >
        <path stroke="none"
              d="M0 0h24v24H0z"
              fill="none"
        >
        </path>
        <path d="M9 19c-4.3 1.4 -4.3 -2.5 -6 -3m12 5v-3.5c0 -1 .1 -1.4 -.5 -2c2.8 -.3 5.5 -1.4 5.5 -6a4.6 4.6 0 0 0 -1.3 -3.2a4.2 4.2 0 0 0 -.1 -3.2s-1.1 -.3 -3.5 1.3a12.3 12.3 0 0 0 -6.2 0c-2.4 -1.6 -3.5 -1.3 -3.5 -1.3a4.2 4.2 0 0 0 -.1 3.2a4.6 4.6 0 0 0 -1.3 3.2c0 4.6 2.7 5.7 5.5 6c-.6 .6 -.6 1.2 -.5 2v3.5">
        </path>
      </svg>
    </a>
  </div>
  <div class="languages">
    <a href="/"
       class="anchor disabled"
       title
    >
      CS
    </a>
    <a href="/"
       class="anchor "
       title
    >
      EN
    </a>
  </div>
  <div class="keys"
       data-test="component-keys"
       style="opacity:1"
  >
    <button type="button"
            class="key disabled"
            aria-label="Předchozí stránka"
            data-test="component-key"
            disabled
    >
      <i class="icon up">
      </i>
    </button>
    <div class="key">
      <button type="button"
              class="key disabled"
              aria-label="Předchozí projekt"
              data-test="component-key"
              disabled
      >
        <i class="icon left">
        </i>
      </button>
      <button type="button"
              class="key "
              aria-label="Následující stránka"
              data-test="component-key"
      >
        <i class="icon down">
        </i>
      </button>
      <button type="button"
              class="key disabled"
              aria-label="Následující projekt"
              data-test="component-key"
              disabled
      >
        <i class="icon right">
        </i>
      </button>
    </div>
  </div>
  <span class="scroll"
        style="opacity:1"
  >
    Scroll
    <i class="dot"
       style="opacity:1;transform:none"
    >
    </i>
  </span>
</footer>
`;
